builders/interoperability/protocols/hyperlane.md

```diff
@@ -21,15 +21,15 @@ The Hyperlane APIs provide a rich suite for developing Web3 applications, ensuri
 
 There are a couple of resources to get you started building cross-chain applications with Hyperlane:
 
-- **[Developer documentation](https://docs.hyperlane.xyz/hyperlane-docs-1/introduction/readme){target=\_blank}** - for technical guides
+- **[Developer documentation](https://docs.hyperlane.xyz/docs/intro){target=\_blank}** - for technical guides
 - **[Hyperlane Explorer](https://explorer.hyperlane.xyz/){target=\_blank}** - to track cross-chain transfers
 
 ## Contracts {: #contracts }
 
 See the list of Hyperlane contracts deployed to Moonbeam, and the networks connected to Moonbeam through Hyperlane.
 
-- **MainNet Contracts** - [Moonbeam](https://docs.hyperlane.xyz/hyperlane-docs-1/developers-faq-and-troubleshooting/addresses#mainnet){target=\_blank}
+- **MainNet Contracts** - [Moonbeam](https://docs.hyperlane.xyz/docs/reference/contract-addresses){target=\_blank}
 
-- **TestNet Contracts** - [Moonbase Alpha](https://docs.hyperlane.xyz/hyperlane-docs-1/developers-faq-and-troubleshooting/addresses#testnet2){target=\_blank}
+- **TestNet Contracts** - [Moonbase Alpha](https://docs.hyperlane.xyz/docs/reference/contract-addresses){target=\_blank}
```

Comment:
This seems weird since they both link to the same place. Maybe we can reformat this to just say view all mainnet and testnet contracts

Reply:
agreed, done